_posts/2021-06-23-cockpit-247.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 247
+author: kkoukiou
+date: 2021-06-23
+tags: cockpit linux
+slug: cockpit-247
+category: release
+summary: Cockpit with Metrics and Virtual Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 247 and Cockpit Machines version 246.
+
+## Metrics: Grafana client setup
```

Comment:
please add some verb here, like "enable Grafana..." or so, or phrase it rather like "new support for ...".  the heading should describe what changed.